src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -727,6 +740,22 @@ Copyright (c) .NET Foundation. All rights reserved.
       </_RestoreGraphEntry>
     </ItemGroup>
 
+    <!-- Non-NuGet type -->
```

Comment:
There's already an ItemGroup above this for a ProjectSpec for packages.config, doesn't this need to be part of that one instead of a duplicate?

Reply:
This should be fixed now :)